code/game/objects/items/robot/robot_items.dm

```diff
@@ -582,6 +582,10 @@
 	process_recharge()
 	process_usage()
 	update_location()
+	if(isliving(host))

```

Comment:
If cyborgs don't disable their modules on death, they should. Make this generic hook please.